### PR TITLE
Location picker frontend

### LINKED ISSUE #127: Implement Location Picker (frontend)
**Branch:** `location-picker-frontend`

### Task at Hand

1. View the [mid-fidelity designs](https://www.figma.com/file/58CoTkFT4TgalsOyc1Patl/%F0%9F%8C%BE-Lagos-Food-Bank?type=design&node-id=1809-21467&mode=design&t=QD3MBT6azmzjOBjZ-0) for the Create Event pages.
2. Open the `LocationPicker` component.
3. Implement the component with the acceptance criteria below.
4. Confirm that your code and pull request follow the [Pull Request Checklist](https://www.notion.so/h4i/Pull-Request-Checklist-910d5e6925df49569067f00f6aa0c0f3).

### Acceptance Criteria

- [ ] The `LocationPicker` component allows you to autocomplete and select locations, like you would on the search bar in Google Maps. This should be then stored as a string value.

### Notes

- Look into [react-google-maps/api](https://www.npmjs.com/package/@react-google-maps/api) for some ideas on using the Google Maps API

### PR DESCRIPTION
**Pull request by Y1chenYao**
## Summary
- implement location picker with google api autocompletion integrated
- closes #127 

## Testing
- run `yarn add react-google-autocomplete` install the [package](https://www.npmjs.com/package/react-google-autocomplete#reactgoogleautocomplete)
> actually we might to do this on our end?
- add the provided api key to `.env.local` in frontend
```
NEXT_PUBLIC_GOOGLE_API_KEY =<api_key>
```
- go to `baseurl/events/create`
- click in-person and type to click an addresses
- open console to see the log of the address

image

## Notes & Resources for Full-stack
- The `AutoComplete` component should be treated as a `html <input>` for styling. For more documentation, see the links below.
- Package link: https://www.npmjs.com/package/react-google-autocomplete#reactgoogleautocomplete
- Google api doc: https://developers.google.com/maps/documentation/javascript/place-autocomplete#constrain-place-types

### CODE REVIEW

**Review by akinfelami — CHANGES_REQUESTED** [review]
Solid work on this guys! 

Everything looks fine to me. The only thing is make sure to add the `package.json` for the frontend folder. It will include the new entry for react-autocomplete... so that other devs can just install it

**Comment by Y1chenYao** [comment]
Fild added!

File: frontend/package.json
Comment (DannyThorne01) [inline frontend/package.json:1]: Is this `package.json` required in the PR?

**Review by DannyThorne01 — COMMENTED** [review]
Great Work!!

**Review by akinfelami — APPROVED** [review]

File: frontend/package.json
Comment (akinfelami) [reply frontend/package.json:1]: Yep! `react-google-autocomplete` was added to the dependencies

**Review by DannyThorne01 — APPROVED** [review]

**Review by louisvalencia — APPROVED** [review]
Very clean code and the feature works as expected! woooo 🕺🏻

**Review by jasozh — APPROVED** [review]

**Pull request merged**